src/Perl6/Metamodel/DefiniteHOW.nqp

```diff
@@ -2,14 +2,28 @@ class Perl6::Metamodel::DefiniteHOW
     does Perl6::Metamodel::Documenting
     does Perl6::Metamodel::Nominalizable
 {
-    # Use coercive and generic flags as bits of a binary representation of the array index
-    my @archetypes := nqp::list(
-            Perl6::Metamodel::Archetypes.new(:definite, :nominalizable, :!coercive, :!generic),
-            Perl6::Metamodel::Archetypes.new(:definite, :nominalizable, :!coercive, :generic),
-            Perl6::Metamodel::Archetypes.new(:definite, :nominalizable, :coercive, :!generic),
-            Perl6::Metamodel::Archetypes.new(:definite, :nominalizable, :coercive, :generic));
+    # ATypeN are used as parameterization arguments to have a definite report correct archetypes.
+    my class AType0 {
```

Comment:
I did want more descriptive naming, but chose the path which fast and uses no extra entities like a list. But since it makes at least two of us then it is more convincing for me to do it.

Will rework when come back from a mini-vacation.